Makefile.include

```diff
@@ -319,6 +324,7 @@ distclean:
 		echo Running: $(MAKE) TARGET=$$TARG clean; \
 		$(MAKE) TARGET=$$TARG clean; \
 	done
+	rmdir $(BUILD_DIR)
```

Comment:
prob want to make this conditional or use `rm -f`, else this errors when `$(BUILD_DIR)` is not found

Reply:
Yes, good idea. Will have to be -rf